### PR TITLE
Sample for configuring Bigtable connection pool

### PR DESCRIPTION
**Pull request by billyjacobson**
Fixes #issue

> It's a good idea to open an issue first for discussion.

- [ ] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] API's need to be enabled to test (tell us)
- [ ] Environment Variables need to be set (ask us to set them)
- [ ] **Tests** pass:   `mvn clean verify` **required**
- [ ] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: bigtable/hbase/snippets/src/test/java/com/example/bigtable/ConfigureConnectionPoolTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import org.hamcrest.CoreMatchers;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ConfigureConnectionPoolTest {
+
+  private static final String INSTANCE_ENV = "BIGTABLE_TESTING_INSTANCE";
```
Comment (kolea2) [inline bigtable/hbase/snippets/src/test/java/com/example/bigtable/ConfigureConnectionPoolTest.java:31]: inline please to be consistent with `GOOGLE_CLOUD_PROJECT` :)

File: bigtable/hbase/snippets/src/test/java/com/example/bigtable/ConfigureConnectionPoolTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  public static void beforeClass() {
+    projectId = requireEnv("GOOGLE_CLOUD_PROJECT");
+    instanceId = requireEnv("BIGTABLE_TESTING_INSTANCE");
+  }
+
+  @Before
+  public void setupStream() {
+    bout = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(bout));
+  }
```
Comment (lesv) [inline bigtable/hbase/snippets/src/test/java/com/example/bigtable/ConfigureConnectionPoolTest.java:51]: Does this get restored ever?

**Review by lesv — CHANGES_REQUESTED** [review]

File: bigtable/hbase/snippets/src/test/java/com/example/bigtable/ConfigureConnectionPoolTest.java (hunk 2 above)
Comment (billyjacobson) [reply bigtable/hbase/snippets/src/test/java/com/example/bigtable/ConfigureConnectionPoolTest.java:51]: I guess not in this test. Will update

File: bigtable/hbase/snippets/src/test/java/com/example/bigtable/ConfigureConnectionPoolTest.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  public static void beforeClass() {
+    projectId = requireEnv("GOOGLE_CLOUD_PROJECT");
+    instanceId = requireEnv("BIGTABLE_TESTING_INSTANCE");
+  }
+
+  @Test
+  public void testConfigureConnectionPool() {
+    bout = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(bout));
+
```
Comment (lesv) [inline bigtable/hbase/snippets/src/test/java/com/example/bigtable/ConfigureConnectionPoolTest.java:52]: Still need to restore this.

**Review by lesv — APPROVED** [review]

**Pull request merged**